### PR TITLE
Some scripts for doing things with Pantheon

### PR DESCRIPTION
**Pull request by hugovk**
Also:
- Use Travis CI to look for problems in the scripts using ShellCheck. This will run every time a PR is created and pass or fail based on ShellCheck.
- Fix the problems ShellCheck found.

### CODE REVIEW

File: deploy_to_live.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```
Comment (takirami) [inline deploy_to_live.sh:1]: Testing comments.
Also, this seems to only push from test to live. Rename script "deploy_test_to_live.sh"?

**Comment by takirami** [comment]
This is probably some general comment.

File: deploy_to_live.sh (hunk 1 above)
Comment (hugovk) [inline deploy_to_live.sh:1]: Line comment worked.

It's only ever possible to deploy from {dev to live} or from {live to test} with Terminus, so these names are shorter.

Unless we also want a deploy_dev_to_live.sh that just calls those two.

**Comment by hugovk** [comment]
Yes, that's a general comment on the PR.

File: deploy_to_live.sh (hunk 1 above)
Comment (takirami) [inline deploy_to_live.sh:1]: Understood,

**Pull request merged**